src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java

```diff
@@ -130,15 +131,51 @@ public Boolean isKnownParticipant(
         return chronicleService.isKnownParticipant( studyId, participantId );
     }
 
+    @RequestMapping(
+            path = STUDY_ID_PATH + PARTICIPANT_ID_PATH,
+            method = RequestMethod.DELETE
+    )
     @Override
-    public void deleteParticipantAndAllNeighbors(
-            UUID studyId, String participantId, DeleteType deleteType ) {
-        // TODO implement this
+    public Void deleteParticipantAndAllNeighbors(
+            @PathVariable( STUDY_ID ) UUID studyId,
+            @PathVariable( PARTICIPANT_ID ) String participantId,
+            @RequestParam( TYPE ) DeleteType deleteType
+    ) {
+        if ( deleteType == null )
```

Comment:
This is super minor, but for consistency and clarity across the codebase we like to wrap all if statements like this in curly brackets, i.e.
```
if ( deleteType == null ) {
  blah
}
```

But, I think if you call this without passing a DeleteType, the request will fail. I could be wrong, but I'm pretty sure in order to make `type` optional, you need to do 
```
@RequestParam( value = TYPE, required = false )
```

Buuuut, I think it actually *does* make sense for this call to fail if a DeleteType isn't specified, since it's an important distinction and is a decision that should probably be made deliberately. So, my preference would be to leave the @RequestParam as is and just remove the `if (deleteType == null)` bit 

(same below)

</stream of consciousness comment>